generateData/coreDBQueries.js

```diff
@@ -0,0 +1,62 @@
+const hrtime = require('process.hrtime');
+const pgdb = require('./pgdbConnection.js');
+const csdb = require('./csdbConnection.js');
+
+pgdb.connect();
+// pick a database to test (between 'postgres' and 'cassandra')
+const selectedDataBase = 'cassandra';
+const noOfQueries = 1;
+const reviewId = 33;
+
+async function csExecuteOneAtATime(info, timer) {
+  const query = 'SELECT * FROM reviews WHERE id=?;';
+  const options = { prepare: true };
+
+  while (info.counter++ < info.totalLength) {
+    const params = [reviewId];
+    const result = await csdb.execute(query, params, options);
+    if (result.rows[0]) {
+      await console.log(`${selectedDataBase} query took ${hrtime(timer, 's')}seconds`);
+      // return result.rows[0];
+    }
+  }
+}
+
+const getReviews = async (db) => {
+  const csdbPromises = new Array(noOfQueries);
```

Comment:
Minor style comment. I think it's more common to make csdbPromises an empty array and push things to it (line 47).

Reply:
Might even help, 10mil is to big to create. I will have to consider this when reworking it.

Reply:
Why 10M? I'd only set noOfQueries to, say, 20.